xlive/H2MOD/Modules/HudElements/RadarPatch.cpp

```diff
@@ -13,18 +9,11 @@ void RadarPatch() {
 	if (H2IsDediServer)
 		return;
 
-	DWORD& MapHeaderType = *(DWORD*)(H2BaseAddr + 0x47CD68 + 0x14C);
-
-	if (MapHeaderType != 1)
-		return;
-
-	DWORD SharedMapMetaDataPointer = *(DWORD*)(H2BaseAddr + 0x47CD64);
-
-	//Tag : ui\hud\bitmaps\hud_sweeper.bitm
-	const DWORD BitmapPropertyOffset = 0x00D93BA8;		//Property : Bitmap
-	const BYTE BitmapDefinitionOffset = 0x0C;			//Definition : Format Offset
-	const WORD BitmapFormatTypeValue = 0x000A;			//Definition : Format Enum
-
-	WriteValue<WORD>(SharedMapMetaDataPointer + BitmapPropertyOffset + BitmapDefinitionOffset, BitmapFormatTypeValue);
+	/* g_D3DDevice9->SetRenderState( D3DRENDERSTATE_DESTBLEND, D3DBLEND_SRCALPHA );
```

Comment:
commented code unnecessary?

Reply:
thats game code to show what i have changed